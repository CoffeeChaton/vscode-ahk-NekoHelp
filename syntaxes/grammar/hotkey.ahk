;@ahk-neko-format-ignore-start

;cmd, param
hotkey, #x           , toggle_follow
hotkey, +$LButton    , click_through
Hotkey, IfWinActive , WinTitle, WinText
Hotkey, IfWinExist , WinTitle, WinText
Hotkey, If , Expression
Hotkey, If, % FunctionObject


;cmd ,param
hotkey ,#x           , toggle_follow
hotkey ,+$LButton    , click_through
Hotkey ,IfWinActive , WinTitle, WinText
Hotkey ,IfWinExist , WinTitle, WinText
Hotkey ,If , Expression
Hotkey ,If, % FunctionObject

;cmd   ,    param
hotkey   ,  #x           , toggle_follow
hotkey  ,    +$LButton    , click_through
Hotkey   ,     IfWinActive , WinTitle, WinText
Hotkey   ,          IfWinExist , WinTitle, WinText
Hotkey      ,      If , Expression
Hotkey    ,    If, % FunctionObject


;cmd param (without,)
hotkey #x           , toggle_follow
hotkey +$LButton    , click_through
Hotkey IfWinActive , WinTitle, WinText
Hotkey IfWinExist , WinTitle, WinText
Hotkey If , Expression
Hotkey If, % FunctionObject

;cmd           param (without,)
hotkey       #x           , toggle_follow
hotkey    +$LButton    , click_through
Hotkey   IfWinActive , WinTitle, WinText
Hotkey           IfWinExist , WinTitle, WinText
Hotkey      If , Expression
Hotkey      If, % FunctionObject

;@ahk-neko-format-ignore-end