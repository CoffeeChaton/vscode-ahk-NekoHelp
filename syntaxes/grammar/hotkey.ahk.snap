>;@ahk-neko-format-ignore-start
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^^^^^^^^ source.ahk comment.line.semicolon.ahk
#                         ^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
>
>;cmd, param
#^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>hotkey, #x           , toggle_follow
#^^^^^^ source.ahk meta.command.key.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^ source.ahk meta.command.key.hotkey.command.ahk
#        ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#         ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk markup.bold.hotkey.command.ahk
#          ^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk
#                     ^ source.ahk meta.command.key.hotkey.command.ahk punctuation.ahk
#                      ^^^^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
>hotkey, +$LButton    , click_through
#^^^^^^ source.ahk meta.command.key.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^ source.ahk meta.command.key.hotkey.command.ahk
#        ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#         ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#          ^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk markup.bold.hotkey.ahk
#                 ^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk
#                     ^ source.ahk meta.command.key.hotkey.command.ahk punctuation.ahk
#                      ^^^^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
>Hotkey, IfWinActive , WinTitle, WinText
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^ source.ahk meta.command.if.hotkey.command.ahk
#        ^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#                   ^ source.ahk meta.command.if.hotkey.command.ahk
#                    ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                     ^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                              ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                               ^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey, IfWinExist , WinTitle, WinText
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^ source.ahk meta.command.if.hotkey.command.ahk
#        ^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#                  ^ source.ahk meta.command.if.hotkey.command.ahk
#                   ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                    ^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                             ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                              ^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey, If , Expression
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^ source.ahk meta.command.if.hotkey.command.ahk
#        ^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#          ^ source.ahk meta.command.if.hotkey.command.ahk
#           ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#            ^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey, If, % FunctionObject
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^ source.ahk meta.command.if.hotkey.command.ahk
#        ^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#          ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#           ^ source.ahk meta.command.if.hotkey.command.ahk
#            ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.definition.variable.percent.ahk
#             ^^^^^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>
>
>;cmd ,param
#^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>hotkey ,#x           , toggle_follow
#^^^^^^ source.ahk meta.command.key.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^ source.ahk meta.command.key.hotkey.command.ahk
#        ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#         ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk markup.bold.hotkey.command.ahk
#          ^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk
#                     ^ source.ahk meta.command.key.hotkey.command.ahk punctuation.ahk
#                      ^^^^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
>hotkey ,+$LButton    , click_through
#^^^^^^ source.ahk meta.command.key.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^ source.ahk meta.command.key.hotkey.command.ahk
#        ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#         ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#          ^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk markup.bold.hotkey.ahk
#                 ^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk
#                     ^ source.ahk meta.command.key.hotkey.command.ahk punctuation.ahk
#                      ^^^^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
>Hotkey ,IfWinActive , WinTitle, WinText
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^ source.ahk meta.command.if.hotkey.command.ahk
#        ^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#                   ^ source.ahk meta.command.if.hotkey.command.ahk
#                    ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                     ^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                              ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                               ^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey ,IfWinExist , WinTitle, WinText
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^ source.ahk meta.command.if.hotkey.command.ahk
#        ^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#                  ^ source.ahk meta.command.if.hotkey.command.ahk
#                   ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                    ^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                             ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                              ^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey ,If , Expression
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^ source.ahk meta.command.if.hotkey.command.ahk
#        ^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#          ^ source.ahk meta.command.if.hotkey.command.ahk
#           ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#            ^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey ,If, % FunctionObject
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^ source.ahk meta.command.if.hotkey.command.ahk
#        ^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#          ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#           ^ source.ahk meta.command.if.hotkey.command.ahk
#            ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.definition.variable.percent.ahk
#             ^^^^^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>
>;cmd   ,    param
#^^^^^^^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>hotkey   ,  #x           , toggle_follow
#^^^^^^ source.ahk meta.command.key.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
#            ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#             ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk markup.bold.hotkey.command.ahk
#              ^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk
#                         ^ source.ahk meta.command.key.hotkey.command.ahk punctuation.ahk
#                          ^^^^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
>hotkey  ,    +$LButton    , click_through
#^^^^^^ source.ahk meta.command.key.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
#             ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#              ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#               ^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk markup.bold.hotkey.ahk
#                      ^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk
#                          ^ source.ahk meta.command.key.hotkey.command.ahk punctuation.ahk
#                           ^^^^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
>Hotkey   ,     IfWinActive , WinTitle, WinText
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#               ^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#                          ^ source.ahk meta.command.if.hotkey.command.ahk
#                           ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                            ^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                                     ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                                      ^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey   ,          IfWinExist , WinTitle, WinText
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^^^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                    ^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#                              ^ source.ahk meta.command.if.hotkey.command.ahk
#                               ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                                ^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                                         ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                                          ^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey      ,      If , Expression
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                   ^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#                     ^ source.ahk meta.command.if.hotkey.command.ahk
#                      ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                       ^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey    ,    If, % FunctionObject
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#               ^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#                 ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                  ^ source.ahk meta.command.if.hotkey.command.ahk
#                   ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.definition.variable.percent.ahk
#                    ^^^^^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>
>
>;cmd param (without,)
#^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>hotkey #x           , toggle_follow
#^^^^^^ source.ahk meta.command.key.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^ source.ahk meta.command.key.hotkey.command.ahk
#       ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#        ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk markup.bold.hotkey.command.ahk
#         ^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk
#                    ^ source.ahk meta.command.key.hotkey.command.ahk punctuation.ahk
#                     ^^^^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
>hotkey +$LButton    , click_through
#^^^^^^ source.ahk meta.command.key.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^ source.ahk meta.command.key.hotkey.command.ahk
#       ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#        ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#         ^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk markup.bold.hotkey.ahk
#                ^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk
#                    ^ source.ahk meta.command.key.hotkey.command.ahk punctuation.ahk
#                     ^^^^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
>Hotkey IfWinActive , WinTitle, WinText
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^ source.ahk meta.command.if.hotkey.command.ahk
#       ^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#                  ^ source.ahk meta.command.if.hotkey.command.ahk
#                   ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                    ^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                             ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                              ^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey IfWinExist , WinTitle, WinText
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^ source.ahk meta.command.if.hotkey.command.ahk
#       ^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#                 ^ source.ahk meta.command.if.hotkey.command.ahk
#                  ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                   ^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                            ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                             ^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey If , Expression
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^ source.ahk meta.command.if.hotkey.command.ahk
#       ^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#         ^ source.ahk meta.command.if.hotkey.command.ahk
#          ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#           ^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey If, % FunctionObject
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^ source.ahk meta.command.if.hotkey.command.ahk
#       ^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#         ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#          ^ source.ahk meta.command.if.hotkey.command.ahk
#           ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.definition.variable.percent.ahk
#            ^^^^^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>
>;cmd           param (without,)
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>hotkey       #x           , toggle_follow
#^^^^^^ source.ahk meta.command.key.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
#             ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#              ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk markup.bold.hotkey.command.ahk
#               ^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk
#                          ^ source.ahk meta.command.key.hotkey.command.ahk punctuation.ahk
#                           ^^^^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
>hotkey    +$LButton    , click_through
#^^^^^^ source.ahk meta.command.key.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^^^ source.ahk meta.command.key.hotkey.command.ahk
#          ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#           ^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk string.2.storage.modifier.hotkey.command.ahk
#            ^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk markup.bold.hotkey.ahk
#                   ^^^^ source.ahk meta.command.key.hotkey.command.ahk meta.hotkey.command.ahk
#                       ^ source.ahk meta.command.key.hotkey.command.ahk punctuation.ahk
#                        ^^^^^^^^^^^^^^ source.ahk meta.command.key.hotkey.command.ahk
>Hotkey   IfWinActive , WinTitle, WinText
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^^ source.ahk meta.command.if.hotkey.command.ahk
#         ^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#                    ^ source.ahk meta.command.if.hotkey.command.ahk
#                     ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                      ^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                               ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                                ^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey           IfWinExist , WinTitle, WinText
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                 ^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#                           ^ source.ahk meta.command.if.hotkey.command.ahk
#                            ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                             ^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#                                      ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                                       ^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey      If , Expression
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#            ^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#              ^ source.ahk meta.command.if.hotkey.command.ahk
#               ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#                ^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>Hotkey      If, % FunctionObject
#^^^^^^ source.ahk meta.command.if.hotkey.command.ahk support.function.hotkey.command.ahk
#      ^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
#            ^^ source.ahk meta.command.if.hotkey.command.ahk keyword.control.flow.hotkey.command.ahk
#              ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.ahk
#               ^ source.ahk meta.command.if.hotkey.command.ahk
#                ^ source.ahk meta.command.if.hotkey.command.ahk punctuation.definition.variable.percent.ahk
#                 ^^^^^^^^^^^^^^^ source.ahk meta.command.if.hotkey.command.ahk
>
>;@ahk-neko-format-ignore-end
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^^^^^^^^ source.ahk comment.line.semicolon.ahk
#                         ^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk